tests/test_models/test_simple_models.py

```diff
@@ -39,29 +39,45 @@ def df():
 def test_simple_model_forecaster_run(simple_df, model):
     sma_model = model()
     sma_model.fit(simple_df)
-    future_ts = simple_df.make_future(future_steps=7)
-    res = sma_model.forecast(future_ts)
+    future_ts = simple_df.make_future(future_steps=7, tail_steps=sma_model.context_size)
+    res = sma_model.forecast(future_ts, prediction_size=7)
     res = res.to_pandas(flatten=True)
     assert not res.isnull().values.any()
     assert len(res) == 14
 
 
+def test_simple_model_forecaster_fail(simple_df):
```

Comment:
Strange behavior, you do the make_future with the context_size tail steps and you still get not enough context

Reply:
We set `window=1000` for `SeasonalMovingAverageModel `, we can't have enough context here.

Reply:
I mean no mistake on the first glance, may be it is better to name the test "test_simple_model_forecaster_fail_not_enough_history"?